exercises/practice/react/lib/react.ex

```diff
@@ -0,0 +1,53 @@
+defmodule React do
+  defmodule InputCell do
+    defstruct [:name, :value]
+    @type t :: %InputCell{name: String.t(), value: any}
+  end
+
+  defmodule OutputCell do
+    defstruct [:name, :inputs, :compute, :value, :callbacks]
+
+    @type t :: %OutputCell{
+            name: String.t(),
+            inputs: [String.t()],
+            compute: (... -> any),
+            value: any,
```

Comment:
If we can allow cells to have any type, should we add a custom test case that tests string cells for example? or boolean cells? (but no error handling of type mismatch)

Problem specifications say:
> If your language supports generics, you may consider allowing reactors to act on other types.
> Tests for that are not included here.

Reply:
Ah, sure, why not! 
Any particular procedure for adding tests?

Reply:
Since those are our own, we don't need to add anything to the toml file. Maybe mention that we did that in `design.md` but otherwise nothing special.